tests/foreman/ui/test_remoteexecution.py

```diff
@@ -239,8 +240,8 @@ def test_positive_run_scheduled_job_template_by_ip(session, module_vm_client_by_
     job_time = 5 * 60
     hostname = module_vm_client_by_ip.hostname
     with session:
-        plan_time = session.browser.get_client_datetime() + datetime.timedelta(seconds=job_time)
         assert session.host.search(hostname)[0]['Name'] == hostname
+        plan_time = session.browser.get_client_datetime() + datetime.timedelta(seconds=job_time)
```

Comment:
Why do we require to change the position of `plan_time` ? any know issue ?

Reply:
Till navigate from Dashboard to host page takes time, I wanted to avoid that causing issues in plan time.